packages/sit-onyx/src/components/OnyxTextarea/OnyxTextarea.vue

```diff
@@ -109,7 +109,11 @@ const handleInput = (event: Event) => {
             v-if="props.required"
             :class="[props.required ? requiredMarkerClass : undefined]"
           ></span>
-          <OnyxInfoTooltip v-if="props.labelTooltip" :text="props.labelTooltip" />
+          <OnyxInfoTooltip
+            v-if="props.labelTooltip"
+            :text="props.labelTooltip"
+            class="onyx-textarea__label-tooltip"
```

Comment:
Do we want to set this inside the `OnyxInfoTooltip` itself instead of classes in every component?

Reply:
found a general solution